storage/s3.go

```diff
@@ -0,0 +1,168 @@
+package storage
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/rapid7/pdf-renderer/cfg"
+)
+
+type s3Client struct {
+	client *s3.S3
+	uploader *s3manager.Uploader
+	downloader *s3manager.Downloader
+	bucket string
+}
+
+
```

Comment:
Clean up

Reply:
Just confirming by "Clean up" you're referring to white space / formatting?

Reply:
yes